scenarios/reproducers/networking-definition.yml

```diff
@@ -91,6 +91,19 @@ cifmw_networking_definition:
               end: 250
       vlan: 23
       mtu: 1500
+    nfs:
```

Comment:
why not call this `storagenfs`?

Reply:
The newly proposed network name does not have "storage" in it since kustomize only supports substring matching and not full regex. Thus, "storagenfs" and "storage" match when they should not and the kustomization does not work. We would see an error like this.
```
  stderr: |-
    The NetConfig "netconfig" is invalid:
    * spec.networks[5].dnsDomain: Invalid value: "storage.example.com": DNSDoman name storage.example.com already in use at spec.networks[3].dnsDomain, must be uniq
    * spec.networks[5].subnets[0].name: Invalid value: "172.18.0.0/24": CIDR 172.18.0.0/24 already in use at spec.networks[3].subnets[0].name
```

Reply:
Thank you for explaining this issue; as you mentioned, we can maintain consistency with other services that have isolated networks named after themselves.. such as `octavia` and `swift` - and call this network `manila`. Acknowledging that `manila` only means `nfs` networking; only used in case where Ceph-NFS is enabled in a deployment.

Reply:
Hi, not that I'm against calling it `manila`, but I see a use case for this network that goes eventually beyond Manila: if glance has a dedicated NFS network (when NFS is the backend), it would result weird attaching a NAD called `manila`. NFS might be generic enough. I see we're only solving Manila and StorageNFS here, so my comment applies only to a long term view where we might need to rename or refactor some code instead of adding an additional composable network.
Thoughts on this?

Reply:
I'm OK either way. I've heard good arguments for both names. @gouthampacha what would you like?

Reply:
@fmount: @fultonj and I were brainstorming and thought that there may not be any other "nfs" consumers... but the potential use case you highlighted seems valid to me as well. Am okay with calling this network "nfs"...